pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Create AsyncAPI Document for applications interacting with Kafka
+description: Explore configuring AsyncAPI document for Kafka messages.
+weight: 20
+---
+
+## Introduction
+
+The tutorial will walk you through the process of creating a schema document for Kafka Messages using AsyncAPI. Additionally, you will also learn about event-driven architecture, pub/sub model, and brokers in relation to Kafka. 
+  
+Let’s assume, you have a service that publishes data to a Kafka topic whenever a new user signs up into the application.
+
+Basically, if you want to build a system that can track the users who have signed up for your application, the entire flow of the application would be:
+
+```mermaid
+sequenceDiagram
+    participant Your Application (Consumer)
+    participant Message Broker
+    participant Service (Producer)
+    participant User
+    Your Application (Consumer)->> Message Broker: Connects to "userSignedUp" topic
+    User-->> Service (Producer): Signs Up
+    Service (Producer)->> Message Broker: Publishes Events
+    Message Broker->> Your Application (Consumer): Sends "user_signup" payload
+```    
+
+
+## Background context
+
+[Event-driven architecture (EDA)](/docs/tutorials/getting-started/event-driven-architectures) is a design pattern that revolves around the production, detection, and reaction to events over time. It consists of three main components: a message broker, event publishers, and subscribers, which together serve as the backbone for event exchange within different services. 
+
+[Message brokers](/docs/tutorials/getting-started/event-driven-architectures#message-broker) facilitate asynchronous communications between services, meaning that the sending service doesn't have to wait for the receiving service's response. This allows multiple services to talk to each other directly, even if they are written in different languages. A great example of such a message broker is Apache Kafka. [Apache Kafka](https://kafka.apache.org/) is a distributed system that is used by thousands of companies for their event-driven system.
+
+[Publish/Subscribe (Pub/Sub)](/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is a typical model in EDAs, providing flexible coupling between publishers and subscribers. In this model, those generating event notifications act as publishers, while those receiving the notifications act as the corresponding subscribers.
```

Comment:
```suggestion
[Publisher/Subscriber (Pub/Sub)](/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is a typical model in EDAs, providing flexible coupling between publishers and subscribers. In this model, those that `send` event notifications act as publishers, while those that `receive` the notifications act as the corresponding subscribers.
```

`Publish/Subscribe` are cursed words here 😄 
also made sure we somehow refer to send/receive that is new